engine.h

```diff
@@ -86,7 +86,7 @@ typedef PtrackMapHdr * PtrackMap;
 /* Block address 'bid' to hash.  To get slot position in map should be divided
  * with '% PtrackContentNblocks' */
 #define BID_HASH_FUNC(bid) \
-		(size_t)(DatumGetUInt64(hash_any_extended((unsigned char *)&bid, sizeof(bid), 0)))
+		(DatumGetUInt64(hash_any_extended((unsigned char *)&bid, sizeof(bid), 0)))
```

Comment:
Yeah, this size_t seems to be not correct, since it is 32 bit wide on 32 bit systems, which is not what we want here. However, next we assign this value to `hash` variable, which again has a `size_t` type, so this change alone does not has much sense, doesn't it? Probably we should change all appearances of `size_t` used for hash map reference to `uint64` for portability?

Reply:
Yep, I recognize `hash` variable as well. I just push-forced branch.